plancton/__init__.py

```diff
@@ -151,6 +151,7 @@ def __init__(self, name, pidfile, logdir, rundir, confdir,
       "docker_privileged" : False,            # run container privileged
       "max_dock_mem"      : 2000000000,       # maximum RAM per container (bytes)
       "max_dock_swap"     : 0,                # maximum swap per container (bytes)
+      "sandbox"           : None,             # directory prefix for sandboxing (with #destination)
```

Comment:
* Use `:` instead of `#`
* Assume `/tmp/plancton-sandboxes` as default prefix
* Always create a sandbox (even if param is `None` or not given)
* Assume `/sandbox` as default inside the container